node/src/chain_spec.rs

```diff
@@ -19,6 +23,14 @@ pub fn get_from_seed<TPublic: Public>(seed: &str) -> <TPublic::Pair as Pair>::Pu
 		.public()
 }
 
+pub fn public_from_ss58<TPublic: Public + FromStr>(ss58: &str) -> TPublic
+where
+	// what's up with this weird trait bound??
```

Comment:
This trait-bound comment we can also remove. It is correct that we need to put it there.

Reply:
comment removed